Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -61,6 +65,7 @@ public FreezeOnMatchCustomization(Type targetType)
 
             this.TargetType = targetType;
             this.Matcher = matcher;
+            this.Request = this.TargetType;
```

Comment:
Instead of assigning the `request` through a property, this constructor could simply invoke the [new one that accepts the `request` as `object`](https://github.com/AutoFixture/AutoFixture/pull/695/files#r75438371).